dashboard/config/linux/upstream-apparmor-kasan.config

```diff
@@ -9338,17 +9391,17 @@ CONFIG_DEBUG_FS_ALLOW_ALL=y
 # CONFIG_DEBUG_FS_ALLOW_NONE is not set
 CONFIG_HAVE_ARCH_KGDB=y
 # CONFIG_KGDB is not set
-CONFIG_ARCH_HAS_UBSAN_SANITIZE_ALL=y
+CONFIG_ARCH_HAS_UBSAN=y
 CONFIG_UBSAN=y
 # CONFIG_UBSAN_TRAP is not set
 CONFIG_CC_HAS_UBSAN_ARRAY_BOUNDS=y
 CONFIG_UBSAN_BOUNDS=y
 CONFIG_UBSAN_ARRAY_BOUNDS=y
 CONFIG_UBSAN_SHIFT=y
+# CONFIG_UBSAN_SIGNED_WRAP is not set
```

Comment:
@dvyukov  do we want to enable these?

Reply:
Let's ask @kees.
Kees, is it ready to be enabled? This includes: the kernel will not fall apart right away with tons of warnings + you are ready to talk to developers that this is important to fix.

But let's not block on this. We can enable this separately.

Reply:
Not yet; there are about a dozen known false positives that we're fixing now, and then we should be ready to turn it on to find rare stuff.